CHANGELOG.md

```diff
@@ -35,6 +29,14 @@ instructions](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds-clus
 
 ### Changed
 
+- **CUMULUS-3245**
+  - Update `@cumulus/lzards-backup` task to either respect the `lzards_provider`
+    deploy-level configuration value or utilize `lzardsProvider` as part of the task
+    workflow configuration
+  -
+  - Minor refactor of `@cumulus/lzards-api-client` to:
+   - Use proper ESS import for `@cumulus/launchpad-auth`
```

Comment:
Did you mean ES6?

Reply:
That got terribly mangled.   Thanks for catching it: bcdbfb01f6